### ISSUE TITLE
Protontricks on Steam Deck doesn't open Wine cmd line or command line for debugging.

### ISSUE THREAD

**Issue by kdliu86** [Matoking/protontricks#244]
**Describe the bug**
Protontricks on Steam Deck doesn't open Wine cmd line or command line for debugging.

**To Reproduce**
1. Run protontricks --gui
2. Select game, select default wine prefix.
3. Try to select Run a Wine cmd shell or Run a commandline shell
(Nothing happens)

**Expected behavior**
Open a command line

**System (please complete the following information):**
 - Distro: Steam OS Latest, Steam Deck Stable
 - Protontricks installation method: Discover store
 - Protontricks version: 1.10.3
 - Steam version: 1691097434

**Additional context**

Other functions in the GUI work fine as far as I can tell.  I can also click Run taskmgr, and start a cmd.exe shell from the task manager.

https://i.imgur.com/XEh0kYo.png

After clicking either of the cmd radio buttons and trying to run, I just get an error that says canberra-gtk-module not found, no other info is spat out.

```
Executing w_do_call cmd
            /tmp/winetricks.tTmAD7o0/no_win64_warnings exists, not issuing 64-bit prefix warning
                                                                                                cmd already installed, skipping
                                                                                                                               Gtk-Message: 16:57:23.475: Failed to load module "canberra-gtk-module"

```

protontricks -v 691150 cmd gets me:

```
Using winetricks 20230212 (protontricks) - sha256sum: 033b8483b0317b58adac2a17333b7d14939b41d237c92d129f069362ef087e9e with wine-8.0 (Staging) and WINEARCH=win64
     Executing w_do_call cmd
                            cmd already installed, skipping
                                                           protontricks (INFO): Terminating launcher process 5
                                                                                                              protontricks (INFO): Launcher process terminated
```

**Comment by kdliu86** [comment]
It was explained to me I have a misunderstanding of ProtonTricks; as the UI is part of winetricks, and these buttons don't reflect protontricks capabilities.

**Comment by Matoking** [comment]
I've pushed an update to the Flatpak version of Protontricks that should go live in a few hours. Both Winetricks commands should work afterwards.

In further detail, the Flatpak version of Protontricks didn't bundle a terminal emulator, meaning a terminal couldn't be opened; `xterm` is now bundled. There was also a bug in Winetricks that caused a crash when the path to the Wine executable contains a space; I've submitted an upstream bug fix at Winetricks/winetricks#2108, though I've also bundled the fix in the Flatpak.
